augur/export_v2.py

```diff
@@ -859,6 +859,13 @@ def _transfer_num_date(node, raw_data):
         if is_valid(raw_data.get("num_date", None)): # it's ok not to have temporal information
             node["node_attrs"]["num_date"] = {"value": format_number(raw_data["num_date"])}
             node["node_attrs"]["num_date"].update(attr_confidence(node["name"], raw_data, "num_date"))
+            # For tips, transfer information about whether the date was inferred or known
+            # and if it was inferred then store the underlying (ambiguous) date string
+            if len(node.get('children', []))==0 and 'date_inferred' in raw_data:
+                node["node_attrs"]["num_date"]["inferred"] = raw_data['date_inferred']
+                if raw_data['date_inferred'] and raw_data.get('raw_date', False):
```

Comment:
Why the check for `raw_data['date_inferred']` in L866 after it's already been used in L865?

Reply:
It can be `false` (not inferred, i.e. known) and if so then `date==raw_date` so we don't export the raw date string.